dts/msm8916/msm8939-qrd-skuk.dts

```diff
@@ -9,9 +9,17 @@
 	qcom,board-id = <0x1000b 5>;
 
 	bq-piccolo {
-		model = "BQ Aquaris M5";
-		compatible = "bq,piccolo", "qcom,msm8939", "lk2nd,device";
-		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_truly_nt35695_1080p_cmd:1:none";
+		model = "BQ Aquaris M5 (Longcheer L9100)";
+		compatible = "longcheer,l9100", "qcom,msm8939", "lk2nd,device";
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "longcheer,l9100-panel";
+
+			qcom,mdss_dsi_truly_nt35695_1080p_cmd {
```

Comment:
Please also add entries for the other possible panels, see https://github.com/msm8916-mainline/android_kernel_qcom_msm8916/blob/901d0b2ca05c0b0af49786fb548eb0fc289867ff/arch/arm/boot/dts/qcom/piccolo.dtsi#L13-L16

They should be added here but there is no need to add them to linux-panel-drivers yet since we have no way of knowing if the panel drivers will actually work correctly.